book/src/help_vc.md

```diff
@@ -221,6 +221,9 @@ Flags:
           automatically enabled for <= 64 validators. Enabling this metric for
           higher validator counts will lead to higher volume of prometheus
           metrics being collected.
+      --graffiti-append
+          When used, client version info will be automatically appended to user
+          custom graffiti.
```

Comment:
We should probably have a note that this should only be used with a Lighthouse BN (due to using a non-standard query parameter).

Reply:
Good catch, makes sense!